dns/rcode.py

```diff
@@ -19,6 +19,7 @@
 
 import dns.enum
 import dns.exception
+import dns.message
```

Comment:
This is creating a circular dependency

Reply:
Removed.

I am not sure if it would be better to add a field to each Exception class with the preferred rcode, but this would require all exception modules to import `rcode` as well.